cogs/ReactForRole.py

```diff
@@ -217,6 +219,146 @@ async def rfr_edit_title(self, ctx: commands.Context):
         else:
             await ctx.send("Okay, cancelling command.")
 
+    @commands.check(KoalaBot.is_admin)
+    @commands.check(rfr_is_enabled)
+    @edit_group.command(name="thumbnail", aliases=["image", "picture"])
+    async def rfr_edit_thumbnail(self, ctx: commands.Context):
+        """
+        Edit the thumbnail of an existing rfr message. User is prompted for rfr message channel ID/name/mention, rfr
+        message ID/URL, new thumbnail, Y/N confirmation. User needs admin perms
+        :param ctx: Context of the command
+        :return:
+        """
+        await ctx.send("Okay, this will edit the thumbnail of a react for role message. I'll need some detail"
+                       "s first though.")
```

Comment:
Just to make it easier to view, please don't split mid word
```suggestion
        await ctx.send("Okay, this will edit the thumbnail of a react for role message. I'll need some "
                       "details first though.")
```